cms/wagtail_hooks.py

```diff
@@ -0,0 +1,47 @@
+"""Custom hooks to configure wagtail behavior"""
+from wagtail.admin.api.views import PagesAdminAPIViewSet
+from wagtail.core import hooks
+
+COURSE_READABLE_ID_FIELD = "{prefix}coursepage__course__readable_id"
+TITLE_FIELD = "title"
+DEFAULT_ORDER = (
+    COURSE_READABLE_ID_FIELD.format(prefix=""),
+    TITLE_FIELD,
+)
```

Comment:
I don't think we need to define 3 different constant values without a possible reuseability, Probably having just one `DEFAULT_ORDER` seems fine and you can set the other values inside it.
 Also could you rename `DEFAULT_ORDER` to something like `DEFAULT_COURSE_LIST_ORDER`?